client/js/app/components/explorer/query_builder/extraction_properties_filter.js

```diff
@@ -0,0 +1,45 @@
+var React = require('react');
+var ReactMultiSelect = require('../../common/react_multi_select.js');
+var ExplorerActions = require('../../../actions/ExplorerActions');
+
+var ExtractionPropertiesFilter = React.createClass({
+
+  _getKeys: function() {
+    var keys = _.keys(this.props.result);
+    var keyList = _.map(keys, function(key) {
```

Comment:
Where is `_.keys`, `_.map` etc coming from in this file? I don't see `lodash` required in the file. Also, we probably shouldn't use `lodash`. We have `es2015` babel in this project so we can use `Object.keys`, `map`, `filter`, etc.